onert-micro/onert-micro/include/execute/OMTestUtils.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ONERT_MICRO_EXECUTE_TESTUTILS_H
+#define ONERT_MICRO_EXECUTE_TESTUTILS_H
+
+#include "test_models/TestDataBase.h"
+#include "OMInterpreter.h"
+
+#include <type_traits>
+
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+namespace onert_micro
+{
+namespace execute
+{
+namespace testing
+{
+
+// Array version of `::testing::FloatNear` matcher.
+::testing::Matcher<std::vector<float>> FloatArrayNear(const std::vector<float> &values,
+                                                      float max_abs_error = 1.0e-5f);
+
+template <typename T, typename U = T>
+std::vector<U> checkSISOKernel(onert_micro::test_model::TestDataBase<T, U> *test_data_base)
```

Comment:
Perhaps it would be better to combine the verification of kernels with one or two inputs (checkSISOKernel and checkTISOKernel) into one template function with a choice of the number of inputs

Reply:
Thank you for review! Yes, I agree, I moved this num of imputs as a functions `checkKernel` argument